### PR TITLE
CFY-7545 new cloudify-rest-service RPM

### PR DESCRIPTION
**Pull request by funkyHat**

### CODE REVIEW

File: rest-service/setup.py [hunk 1]
```diff
@@ -73,4 +73,8 @@
     license='LICENSE',
     description='Cloudify manager rest service',
     zip_safe=False,
-    install_requires=install_requires)
+    install_requires=install_requires,
+    extras_require={
+        'dbus': ['dbus-python==1.2.4'],
```
Comment (mcouthon) [inline rest-service/setup.py]: What's that for?

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,149 @@
+version: 2
```
Comment (mcouthon) [inline .circleci/config.yml]: Does this new file remove the need for `run-tests.sh`?

File: packaging/cloudify-rest-service.spec [hunk 3]
```diff
@@ -84,7 +84,7 @@ systemd-tmpfiles --create
 /etc/sudoers.d/cloudify-restservice
 /opt/restservice/NOTICE.txt
 /opt/restservice/set-manager-ssl.py*
-/usr/lib/systemd/system/cloudify-rest-service.service
+/usr/lib/systemd/system/cloudify-restservice.service
```
Comment (mcouthon) [inline packaging/cloudify-rest-service.spec]: Probably applies to the logrotate file as well?

File: rest-service/setup.py (hunk 1 above)
Comment (tehasdf) [reply rest-service/setup.py]: dbus-python is used for `cfy status`; currently we do a couple `ln` calls to use the systemwide install of it, but it is nicer to simply install it into the venv instead
(current: https://github.com/cloudify-cosmo/cloudify-manager-install/blob/09472e7b91f98873925ba88ef9590cbca5ace540/cfy_manager/components/restservice/restservice.py#L102 )

File: .circleci/config.yml (hunk 2 above)
Comment (funkyHat) [reply .circleci/config.yml]: yes

File: packaging/cloudify-rest-service.spec (hunk 3 above)
Comment (funkyHat) [reply packaging/cloudify-rest-service.spec]: tbh I'd rather have them matching the package name as much as possible. This one has to be like this or else requires a bunch of changes elsewhere (cfy status etc.)
The name of the logrotate file is essentially arbitrary

**Comment by tehasdf** [comment]
The `upload_to_s3.py` stuff was moved out to #1098

**Comment by tehasdf** [comment]
the upload_to_s3 stuff was moved back in here after all ;p

**Pull request merged**